.github/actions/security/bandit/action.yaml

```diff
@@ -111,31 +111,50 @@ runs:
     - name: Run Bandit scan
       id: run-bandit
       shell: bash
+      env:
+        IN_SCOPE: ${{ inputs.scan-scope }}
+        IN_PATHS: ${{ inputs.paths }}
+        IN_CONFIG: ${{ inputs.config_file }}
+        IN_SEV_LVL: ${{ inputs.severity-level }}
+        IN_CONF_LVL: ${{ inputs.confidence-level }}
+        IN_OUTPUT: ${{ inputs.output-format }}
+        IN_FAIL: ${{ inputs.fail-on-findings }}
```

Comment:
would it be possible to rename these and use the full version as the variable name instead of abbreviations. It could be easier to remember?

Like `INPUT_SEVERITY_LEVEL` etc? It's more verbose but easier to read and follow